src/runtime/com.rs

```diff
@@ -0,0 +1,44 @@
+use crate::*;
+
+/// Initializes COM for use by the calling thread for the multi-threaded apartment (MTA).
+pub fn initialize_mta() -> Result<()> {
```

Comment:
Aren’t the names of the functions backwards? This initializes for a single threaded apartment but is called `mta`

Reply:
Yes, the comments are backwards. I'll switch to constants. 😉